larray/viewer/model.py

```diff
@@ -118,15 +119,11 @@ class ArrayModel(QAbstractTableModel):
 
     Parameters
     ----------
-    data : 2D NumPy array, optional
-        Input data (2D array).
+    data : LArray or NumPy array, optional
```

Comment:
or dataframe, or any array-like? actually anything which can be converted to an LArray